tests/baselines/reference/tsxAttributeResolution12.errors.txt

```diff
@@ -46,15 +46,15 @@ tests/cases/conformance/jsx/file.tsx(28,11): error TS2322: Type '{}' is not assi
     const T = TestMod.Test;
     var t1 = <T />;
               ~
-!!! error TS2322: Type '{}' is not assignable to type 'IntrinsicAttributes & { reqd: any; }'.
-!!! error TS2322:   Type '{}' is not assignable to type '{ reqd: any; }'.
-!!! error TS2322:     Property 'reqd' is missing in type '{}'.
+!!! error TS2610: The attributes provided to T are not assignable to type 'IntrinsicAttributes & { reqd: any; }'

```

Comment:
See what's going on here (unrelated)

Reply:
The second change in this test also shows off that we need to print entitynames, not just identifiers.

Reply:
Looks like IntrinsicAttributes is defined, but IntrinsicClassAttributes isn't. The predicate needs to give up a little less easily.